### PR TITLE
adding a line

### PR DESCRIPTION
**Pull request by SnehaDL**
testing

### CODE REVIEW

**Comment by snehadesoftech** [comment]
please add a comment before the code

File: test.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Program to check if a string
 #  is palindrome or not
```
Comment (snehadesoftech) [inline test.py]: - [ ] please
  
  add a comment

**Comment by SnehaDL** [comment]
please add a comment

**Comment by SnehaDL** [comment]
done

**Pull request merged**